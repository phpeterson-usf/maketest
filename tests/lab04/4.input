-e "~0b11111111 + 1" -b 2
