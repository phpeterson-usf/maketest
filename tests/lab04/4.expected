0b11111111111111111111111100000001
